proprietary/tokens/src/components/utrecht/textbox.tokens.json

```diff
@@ -0,0 +1,72 @@
+{
+  "utrecht": {
+    "textbox": {
+      "background-color": {},
+      "border-bottom-width": {},
+      "border-color": {
+        "value": "{amsterdam.color.primary-black}"
+      },
+      "border-radius": {},
+      "border-width": {},
+      "color": {
+        "value": "{amsterdam.color.primary-black}"
+      },
+      "font-family": {
+        "value": "{amsterdam.typography.font-family}"
+      },
+      "font-size": {
+        "value": "{amsterdam.typography.font-size.24}"
+      },
+      "line-height": {},
```

Comment:
`line-height` should be set as well, but maybe we can do that after we implement the text levels